Neos.Eel/Classes/Helper/ArrayHelper.php

```diff
@@ -342,15 +342,21 @@ public function pop(iterable $array): array
      *
      *     Array.push(array, e1, e2)
      *
-     * @param iterable $array
+     * @param iterable|scalar $array
      * @param mixed $element
      * @return array The array with the inserted elements
      */
-    public function push(iterable $array, $element): array
+    public function push($array, $element): array
     {
         if ($array instanceof \Traversable) {
             $array = iterator_to_array($array);
         }
+        if (is_scalar($array)) {
```

Comment:
With `is_scalar` it would not be possible to push objects like `node` or resources, right?

Reply:
see comment above ^^ 
https://github.com/neos/flow-development-collection/pull/2733#discussion_r829944067